HPXMLtoOpenStudio/tests/test_constructions.rb

```diff
@@ -13,6 +13,35 @@ def sample_files_dir
     return File.join(File.dirname(__FILE__), '..', '..', 'workflow', 'sample_files')
   end
 
+  def _get_ems_values(ems_objects, name)
```

Comment:
Since we now have the same (or similar?) methods in multiple test_foo.rb files, can you move the method to util.rb so it can be shared across test files?